src/main/kotlin/lotto/Lottos.kt

```diff
@@ -1,17 +1,22 @@
 package lotto
 
-class Lottos(private val amountOfLotto: Int) {
-    private val lottos = (0 until amountOfLotto).map { Lotto() }
+class Lottos(private val amountOfLotto: Int, private val manualLottos: List<Lotto>) {
+    val lottos = getLottos(amountOfLotto, manualLottos)
 
-    fun getLottos() = lottos
+    fun getLottos(amountOfLotto: Int, manualLottos: List<Lotto>): List<Lotto> {
+        val lottos = manualLottos.toMutableList()
+        val automaticLottoCount = amountOfLotto - manualLottos.size
+        for (number in 0 until automaticLottoCount) {
+            lottos.add(Lotto(LottoProgram.getRandomNumbers()))
+        }
+        return lottos.toList()
```

Comment:
MutableList 생성, add(), List 반환이 하나의 함수에서 일어나면 map 함수로 대체할 수 있습니다.

```suggestion
        val automaticLottoCount = amountOfLotto - manualLottos.size
        val autoLottos = (1..automaticLottoCount)
            .map { Lotto(LottoProgram.getRandomNumbers()) }
        return manualLottos + autoLottos
```